src/libraries/System.Private.Xml.Linq/tests/xNodeReader/CXMLReaderAttrTest.cs

```diff
@@ -532,7 +531,15 @@ public void NegativeOneOrdinal()
                 {
                     XmlReader DataReader = GetReader();
                     PositionOnElement(DataReader, "ACT1");
-                    string str = DataReader.GetAttribute(-1);
+
```

Comment:
Please add new .cs file and add Xunit [Fact] for new tests (add failing test first to ensure it's executed). The old tests are badly designed and we should add new tests better way and hopefully convert old tests to new style somewhere in the future.

Reply:
I didn't initially noticed you removed the line here. I'm fine with leaving this here but please use Assert.Throws